antsichaut/antsichaut.py

```diff
@@ -474,6 +485,7 @@ def main() -> None:
         {"title": "removed_features", "labels": args.removed_features_labels},
         {"title": "security_fixes", "labels": args.security_fixes_labels},
         {"title": "bugfixes", "labels": args.bugfixes_labels},
+        {"title": "doc_changes", "labels": args.doc_changes_labels},
         {"title": "skip_changelog", "labels": args.skip_changelog_labels},
```

Comment:
Is this adding a `docs_changes` section to the `changelogs/changelog.yaml` file? This would be problematic since `changelogs/config.yaml` likely doesn't know about it, and such a section name also isn't standardized (see the specs https://github.com/ansible-community/antsibull-changelog/blob/main/docs/changelog.yaml-format.md).

Reply:
@felixfontein this has been added considering ansible network collections, [ref](https://github.com/ansible-collections/cisco.ios/blob/f95c729cd4cdfa88f4aeb3c500bda06b19badc5f/changelogs/config.yaml#L29)
which considers doc_changes as a valid changelog attribute.

Reply:
Should a PR be added in core for this to be a valid part of the schema?

Reply:
This is unrelated to core, it's rather a community issue. It should probably be brought up in the docs working group (DaWGs).

Also since most collections do *not* support this (with an explicit config file that does not cover this section), this should probably be something that's configurable per collection for antsichaut (probably a good idea anyway to make the labels configurable), or antsichaut should use the antsibull-changelog config.